### PR TITLE
Cosmetic: Add in_review.py script

### PR DESCRIPTION
**Pull request by willkg**
Adds in-review script that I use for quarterly and yearly rollups for Input.

r?

### CODE REVIEW

File: scripts/in_review.py [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+import subprocess
+import sys
+import textwrap
+import urllib
+
+import requests
+
+
+BUGZILLA_API_URL = 'https://bugzilla.mozilla.org/rest/'
+BUGZILLA_PRODUCT = 'Input'
```
Comment (mythmon) [inline scripts/in_review.py]: Uh...

File: scripts/in_review.py [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+        # FIXME - this is lame. what's going on is that there are
+        # merge commits which have multiple authors, so we just grab
+        # the second one.
+        if '\n' in author:
+            author = author.splitlines()[1]
+
+        committers[author] = committers.get(author, 0) + 1
+        all_people.add(author)
+
+        diff_data = git('git', 'diff', '--numstat', '--find-copies-harder',
```
Comment (mythmon) [inline scripts/in_review.py]: Ha. I forgot about `--find-copies-harder`.

**Comment by mythmon** [comment]
r+ with the right product.

File: scripts/in_review.py (hunk 1 above)
Comment (rehandalal) [inline scripts/in_review.py]: yes me just had this same problem

Comment (willkg) [inline scripts/in_review.py]: You two are *so* demanding.

**Comment by willkg** [comment]
^^^ Fixed.

**Comment by mythmon** [comment]
Travis can't possibly have a problem with this.

**Pull request merged**